content/articles/computer-network-services/index.md

```diff
@@ -0,0 +1,194 @@
+### Understanding computer network services
+#### Introduction
+
+Network service is an operand in the network provided by a server. The application runs at the network application layer and above, and includes the following;
+ - IP addressing 
+ - Domain Name System (DNS)
+ - Internet access
+ - Security products (firewalls) 
+ - Primary domain service 
+ - Web content filtering 
+ - VPN termination 
+ - Intrusion prevention systems (IPS)
+
+A server component for each service is usually run on one or more computers. These servers are accessed through a group of client computers on the network. These clients and servers are not only coupled with hardware usage but also user interfaces for human-computer interactions.
+
+### Table of contents
+
+1. [Basic Network Design Requirements](#basic-network-design-requirements)
+2. [Scaling a network](#scaling-a-network)
+3. [Network services and their functions](#network-services-and-their-functions)
+4. [Automatic Network Services](#automatic-network-services)
+5. [Configuring Network Services](#configuring-network-services)
+
+
+### Basic Network Design Requirements
+
+1. Security
+
+Security can be achieved through network authentication and authorization, encryption on network
+communications i.e restricting access to only authorized users.
+This ensures confidentiality, integrity, and accountability of data which is vital in a network. A breach in
+security may lead to consequences such as ;
+- Loss of valuable data
+- Lawsuit
+- Soiled reputation
+- Loss of clientele
+- Financial loss
+- Loss of operational permits
+- Interruption of services, etc
+
+2. Scalability
+
+Scalability is the ability of a network to accommodate increasing streams of data traffic and additional
+number of users without degrading performance. It can be achieved through ;
+- High capability devices
+- Proper configuration
+- High bandwidth media
+- Choice of topology
+
+3. Redundancy / Fault tolerance
+
+This is the ability to withstand a hostile set of conditions and continue operation, that is recover from a
+catastrophic network failure within the shortest possible time i.e Mean Time Between Failure (MTBF)
+and Mean Time To Repair (MTTR). It can be achieved through ;
+- Network monitoring
+- Services clustering
+- RAID
+
+4. Quality of Services (QoS)
+
+QoS is a metric measuring the level of distribution of the original product in regards to the quality of the
+transmitted data despite the distance. It can be achieved through data prioritization for devices with the
+capability.
+
+### Scaling a Network
```

Comment:
write all headers in the required format. Please refer to other published articles.